lib/Doctrine/Template/Geographical.php

```diff
@@ -113,7 +113,7 @@ public function getDistance(Doctrine_Record $record, $kilometers = false)
 
         $result = $query->execute()->getFirst();
 
-        if (isset($result['kilometers']) && $result['miles']) {
+        if ($result && isset($result['kilometers']) && $result['miles']) {
```

Comment:
`$result` could be false here (`getFirst` may return `false` if the `Doctrine_Collection` is empty).